client/src/utils/common.ts

```diff
@@ -29,7 +29,7 @@ const getPrettyTime = (hour: number, minute: number, second: number) => {
     return prettyText;
 };
 
-export const getHumanReadableTime = (seconds: number, invalidText = '-') => {
+export const getHumanReadableTime = (seconds?: number, invalidText = '-') => {
     if (!seconds) {
```

Comment:
!seconds will be true for value 0, we should either use ``isTruthy(seconds)`` or  ``seconds === undefined``